zeebe-cluster/values.yaml

```diff
@@ -54,6 +55,7 @@ JavaOpts: >-
 
 labels:
   app: zeebe    
+podLabels: {}
```

Comment:
@caugustus-ezcater can we avoid having these extra empty default values in the `values.yaml` file?